pages/coq-team.html

```diff
@@ -0,0 +1,503 @@
+    <div class="picture"><img src="/files/team/nopic.png" alt="Jim Fehrle" width="200px"/></div>
+    <div class="name">Jim Fehrle</div><div class="nickname">jfehrle</div>
+    <div class="position">Software Engineer in Menlo Park, CA, USA</div>
+    <div class="components">Documentation, Proof diffs, LTac debugger</div>
+  </div>
+  <div class="person">
+    <div class="picture"><img src="/files/team/nopic.png" alt="Julien Forest" width="200px"/></div>
+    <div class="name"><a href="http://web4.ensiie.fr/~forest/">Julien Forest</a></div>
+    <div class="nickname">forestjulien</div>
+    <div class="position">Assistant Professor at the ENSIEE/CNAM, Evry/Paris, France</div>
+    <div class="components">Function</div>
+  </div>
+  <div class="person">
+    <div class="picture">
+      <img src="/files/team/ggonthier.jpg" alt="Georges Gonthier" width="200px"/></div>
+    <div class="name">Georges Gonthier</div><div class="nickname">ggonthier</div>
+    <div class="position">Inria Advanced Researcher in Saclay, France</div>
+    <div class="components">Mathematical Components, Notations</div>
+  </div>
+  <div class="person">
+    <div class="picture"><img src="/files/team/bgregoir.jpg" alt="Benjamin Grégoire" width="170px"/></div>
+    <div class="name"><a href="http://www-sop.inria.fr/members/Benjamin.Gregoire/">Benjamin Grégoire</a></div>
+    <div class="nickname">bgregoir</div>
+    <div class="position">Inria Researcher in the Stamp Team, Sophia-Antipolis, France</div>
+    <div class="components">Ring, <verb>vm_compute</verb></div>
+  </div>
+  <div class="person">
+    <div class="picture"><img src="/files/team/vbgl.jpg" alt="Vincent Laporte" width="160px"/></div>
+    <div class="name"><a href="https://members.loria.fr/VLaporte/">Vincent Laporte</a></div><div class="nickname">vbgl</div>
+    <div class="position">Inria Researcher in the Pesto Team, Nancy, France</div>
+    <div class="role">RM: 8.10</div>
+    <div class="components">Nix, micromega</div>
+  </div>
+  <div class="person">
+    <div class="picture">
+      <img src="/files/team/assia_mahboubi.jpg" alt="Assia Mahboubi" width="200px"/></div>
+    <div class="name"><a href="http://people.rennes.inria.fr/Assia.Mahboubi/">Assia Mahboubi</a></div><div class="nickname">amahboubi</div>
+    <div class="position">Inria Researcher in the Gallinette Team, Nantes, France</div>
+    <div class="role"><a href="https://cordis.europa.eu/project/id/101001995">ERC FRESCO PI</a></div>
+    <div class="components">Ring, Mathematical Components</div>
+  </div>
+  <div class="person">
+    <div class="picture">
+      <img src="/files/team/erikmd.png" alt="Érik Martin-Dorel" width="170px"/>
+    </div>
+    <div class="name"><a href="https://www.irit.fr/~Erik.Martin-Dorel/">Érik Martin-Dorel</a></div><div class="nickname">erikmd</div>
+    <div class="position">Assistant Professor at IRIT, Toulouse University, France</div>
+    <div class="components">Primitive Types, Docker</div>
```

Comment:
May you add Proof-General as well for me?
```suggestion
    <div class="components">Primitive Types, Docker, Proof-General</div>
```